.gitignore

```diff
@@ -5,3 +5,5 @@ result
 test.log
 build.log
 src/config.mlh
+src/external/ocaml-rocksdb/rocksdb
```

Comment:
are we sure we want to ignore? Did you make modifications that are needed?

Reply:
this is the clone from an install script, we do not modify it